curp/src/lib.rs

```diff
@@ -180,5 +184,5 @@ mod rpc;
 /// Snapshot
 mod snapshot;
 
-#[cfg(test)]
-pub(crate) mod test_utils;
+// #[cfg(test)]
```

Comment:
Plz remove these useless comments if we don't need it anymore.

Reply:
done